index.js

```diff
@@ -1,56 +1,87 @@
-let books = [];
-const title = document.querySelector('.title');
-const author = document.querySelector('.author');
-const displayContainer = document.querySelector('.container');
+// const { isClosingBraceToken } = require('eslint-utils');
```

Comment:
- [ ] I would like to recommend you remove all the commented code from your codebase in order to keep your code clean :smile: